src/views/Stake/StakeV2.js

```diff
@@ -912,160 +1302,321 @@ export default function StakeV2({
-                              {formatKeyAmount(vestingData, "mlpVesterVestedAmount", 18, 4, true)} esMYC deposited for
-                              vesting.
-                            </>
-                          );
-                        }}
-                      />
+              <StakeV2Styled.Card className="Staking">
+                <StakeV2Styled.VestingInfo>
+                  <StakeV2Styled.StakingBannerRow>
+                    <div className="App-card-row break">
+                      <div className="label">Wallet</div>
+                      <div>
+                        <StakeV2Styled.FlexRowEnd>
+                          <WalletBalance
+                            walletAmount={userMycBalance || ZERO_BN}
+                            tokenUsdPrice={mycPrice || ZERO_BN}
+                            selectedToken={CompatibleTokenEnum.MYC}
+                          />
+                          <StakeV2Styled.StakingButton onClick={() => stakeToken(MYC_TOKEN, STAKE)}>
+                            Stake
+                          </StakeV2Styled.StakingButton>
+                        </StakeV2Styled.FlexRowEnd>
+                        <StakeV2Styled.FlexRowEnd>
+                          <WalletBalance
+                            walletAmount={userEsMycBalance || ZERO_BN}
+                            tokenUsdPrice={mycPrice || ZERO_BN}
+                            selectedToken={CompatibleTokenEnum.esMYC}
+                          />
+                          <StakeV2Styled.StakingButton onClick={() => stakeToken(ES_MYC_TOKEN, STAKE)}>
+                            Stake
+                          </StakeV2Styled.StakingButton>
+                        </StakeV2Styled.FlexRowEnd>
+                      </div>
                     </div>
-                  </div>
-                  <div className="App-card-row">
-                    <div className="label">Claimable</div>
+                    <StakeV2Styled.Divider />
+                    <div className="App-card-row break">
+                      <div className="label">Staked</div>
+                      <div>
+                        <StakeV2Styled.FlexRowEnd>
+                          <WalletBalance
+                            walletAmount={userStakedMycBalance || ZERO_BN}
+                            tokenUsdPrice={mycPrice || ZERO_BN}
+                            selectedToken={CompatibleTokenEnum.MYC}
+                          />
+                          <StakeV2Styled.StakingButton onClick={() => stakeToken(MYC_TOKEN, UNSTAKE)}>
+                            Withdraw
+                          </StakeV2Styled.StakingButton>
+                        </StakeV2Styled.FlexRowEnd>
+                        <StakeV2Styled.FlexRowEnd>
+                          <WalletBalance
+                            walletAmount={userStakedEsMycBalance || ZERO_BN}
+                            tokenUsdPrice={mycPrice || ZERO_BN}
```

Comment:
I think do this conditional in the WalletBalance component, less duplication, same with walletAmount

Reply:
good idea